editor.html

```diff
@@ -310,9 +310,9 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                         <div class="buttons_menu_item">
                             <span id="lang-select">Select Language:</span>
                             <ul class="tree">
-                                <li><a href="#" class="lang-choice" id="en" tabindex="121">English</a></li>
-                                <li><a href="#" class="lang-choice" id="es" tabindex="122">Spanish</a></li>
-                                <li><a href="#" class="lang-choice" id="pl" tabindex="123">Polish</a></li>
+                                <li><a title="English" href="#" class="lang-choice" id="en" tabindex="121">English</a></li>
+                                <li><span title="Spanish"><a lang="es" href="#" class="lang-choice" id="es" tabindex="122">Español</a></span></li>
+                                <li><span title="Polish"><a lang="pl" href="#" class="lang-choice" id="pl" tabindex="123">Polski</a></span></li>
```

Comment:
I think the `title` entry will have to be translated to each language.
So in English the title is `Polish`, but in Spanish the title would be `Polaco`.
So we'll need new entries in the translation files.